### PR TITLE
removed "obsoletion candidate" tag and obsoletion date

### LINKED ISSUE #5558: remove "obsoletion candidate" tag from obsoleted terms
a lot of obsoleted terms still have the "obsoletion candidate" tag. 
we should remove these tag when the term is obsoleted

### PR DESCRIPTION
**Pull request by sabrinatoro**
Closes #5558

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -453558,7 +453235,6 @@ created_by: https://orcid.org/0000-0001-5208-3432
 id: MONDO:0100282
 name: SC phocomelia syndrome
 comment: Reason: duplicate. This will be merged with MONDO:0100253 Roberts-SC phocomelia syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:453237]: could you please remove these comments too?
thanks!

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
please remove the comments if we are not going to obsolete the terms - thanks!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo]: done

**Comment by sabrinatoro** [comment]
closes #5561

**Review by nicolevasilevsky — APPROVED** [review]
thank you @sabrinatoro!

**Pull request merged**